### PR TITLE
fix: remove unnecessary sort

### PR DESCRIPTION
**Pull request by BrodaNoel**
Fixing tech debt from #124 

While running `pnpm dev` and `pnpm test`, the file `src/__mocks__/__fixtures__/GetAccount-7c2174/5d26cb.json` was created in my local env, but I didn't commit it. It seems to be a full copy of `src/__mocks__/__fixtures__/GetAccount-7c2174/169171.json` (which is tracked on the repo). Let me know if you need me to push it.

The content of the file is:

```json
{
  "accounts": {
    "nodes": [
      {
        "id": 2,
        "idSs58": "cFNzKSS48cZ1xQmdub2ykc2LUc5UZS2YjLaZBUvmxoXHjMMVh"
      },
      {
        "id": 3,
        "idSs58": "cFJXT4WEEdfiShje4z9JMwAvMiMTu7nioPgXsE9o1KqdVrzLg"
      },
      {
        "id": 7,
        "idSs58": "cFLGvPhhrribWCx9id5kLVqwiFK4QiVNjQ6ViyaRFF2Nrgq7j"
      },
      {
        "id": 9,
        "idSs58": "cFKZarxpf9MVwzzmYUtQfV61PRkYgTj9wUgUCeuKpKgMLrTow"
      },
      {
        "id": 27,
        "idSs58": "cFLBKavxvThwqLWNr7cTwtqhYD6jDqXM31d6QoTLvuK4X78ve"
      },
      {
        "id": 34,
        "idSs58": "cFK6qCSmgYJACMNVk6JnCb5nkccr7yM6aZiKtXUnFAzsX7hvs"
      },
      {
        "id": 36,
        "idSs58": "cFKy4xbhLxvAVxYuPEWbbTJTed5WtyqNVikH2fS2WYLNHRrFh"
      },
      {
        "id": 50,
        "idSs58": "cFJtLQCKAGqSpamy1R9W4efQHaq5PhKSK93ucVRZMJq6ute8F"
      },
      {
        "id": 53,
        "idSs58": "cFPV6R6vXWAXVityn3XAZNMQDkuZ96LcmS2euYYrGjchWWAri"
      },
      {
        "id": 56,
        "idSs58": "cFPJNbXH9KNP1CRejnf19ARopcS8w8c4teTz5GF3G36MZRWJG"
      },
      {
        "id": 57,
        "idSs58": "cFNgY2xnF9jvLLJ9TTtFwVTUCoo9aAX26UveiN7NftzkhEyYW"
      },
      {
        "id": 70,
        "idSs58": "cFMVQrUbuTuXmeRinPQovRkCgoyrrRd3N4Q5ZdHfqv4VJi5Hh"
      },
      {
        "id": 79,
        "idSs58": "cFNyp1zp93cBrHnsPjSpgc2JGwGmiQbtrrTNRNeZteRkb3Ud4"
      },
      {
        "id": 103,
        "idSs58": "cFHw1UHP69RiSSj5FSYRx5e4Fuh4qg4625zBxW2XbE4wK6JTL"
      },
      {
        "id": 109,
        "idSs58": "cFJYzUFU97Y849kbKvyj7br1CUumnbqWHJKDcfPFoKRqq6Zxz"
      },
      {
        "id": 118,
        "idSs58": "cFMTFP2F61oRQGf7rUZpD2gaNtEjH65Sm6xhay4xmgBcwD26a"
      },
      {
        "id": 131,
        "idSs58": "cFNAnhKPq7hnCKBM2rgo7v7WyroRsZiKm9gKSoPcggdyfYwaA"
      },
      {
        "id": 153,
        "idSs58": "cFPQQaEEGnzuZTBg8S3JuhiTu1CTrHXDzKAZfvNrSrjUUQQ1z"
      },
      {
        "id": 155,
        "idSs58": "cFLZS9GDX4CeXWdjqm2sXmVUNqW1H71BK5nfUXHo6qtKDqNHu"
      }
    ]
  }
}
```

### CODE REVIEW

File: src/queries/__tests__/lpFills.test.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ describe('getLpFills', () => {
         {
           "alias": "Selini",
           "filledAmountValueUsd": "9434345.0577949431",
-          "idSs58": "cFKZarxpf9MVwzzmYUtQfV61PRkYgTj9wUgUCeuKpKgMLrTow",
+          "idSs58": "cFLGvPhhrribWCx9id5kLVqwiFK4QiVNjQ6ViyaRFF2Nrgq7j",
```
Comment (BrodaNoel) [inline src/queries/__tests__/lpFills.test.ts:26]: After some minutes (~300 minutes 😆) dealing with this weird unexpected-change, I realized that `lpAliasMap` has different keys for the same accounts, like:

```
  cFLGvPhhrribWCx9id5kLVqwiFK4QiVNjQ6ViyaRFF2Nrgq7j: { name: 'Selini', twitter: '@SeliniCapital' },
  cFKZarxpf9MVwzzmYUtQfV61PRkYgTj9wUgUCeuKpKgMLrTow: { name: 'Selini', twitter: '@SeliniCapital' },
  cFLBRkucKofjzUNnFpSoW8Lv4RK24K5tUPFKAWgdrBci9WPDY: { name: 'Selini', twitter: '@SeliniCapital' },
```

So, I'm not 100% sure, but this change seems to be expected, considering that both keys are valid bot the same account.

I wonder if that should actually work in that way, or if we have to keep some main/original key

**Comment by BrodaNoel** [comment]
Let me know if I have to push the mentioned file in order to fix the `test` fail.

```
 FAIL  src/queries/__tests__/lpFills.test.ts > getLpFills > returns all the lp fills for given range
AssertionError: No cache found for query, please run the tests locally first
```

**Comment by acdibble** [comment]
> Let me know if I have to push the mentioned file in order to fix the `test` fail.
> 
> ```
>  FAIL  src/queries/__tests__/lpFills.test.ts > getLpFills > returns all the lp fills for given range
> AssertionError: No cache found for query, please run the tests locally first
> ```

Yes, push it please. It's our automocking for the tests

File: src/queries/__tests__/lpFills.test.ts (hunk 1 above)
Comment (sukhoviianastasia) [reply src/queries/__tests__/lpFills.test.ts:26]: Yes, that's correct—different keys/addresses may share the same alias. As for the test file, we are pushing it along. 👍 Thanks for your observation!

**Comment by BrodaNoel** [comment]
Done. Mock file pushed.

**Review by sukhoviianastasia — APPROVED** [review]

**Review by acdibble — APPROVED** [review]

**Pull request merged**